### ISSUE TITLE
Add readme

### ISSUE THREAD

**Issue by ertrzyiks** [Eluss/NoSimulatorUnitTests#1]
Hey man, 
could you add a readme file with instruction how to run tests or at least link to your blog post?

**Comment by Eluss** [comment]
You're right. I'll link to post right now, but I'll surely add a readme soon.
Thanks for that
